### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by GewoonMaarten**
As discussed in one of our retrospectives.

### CODE REVIEW

**Review by DanielvanVliet — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Additional context
+Add any other context or screenshots about the feature request here.
+
+### Acceptance criteria
+Conditions that must be satisfied to be accepted by a user, customer or other stakeholder.
+- [ ] When I click the Rank button, then ideas are sorted with the top-scoring ideas at the top
```
Comment (bddjong) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]: What's this about?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+### Definition of Done
+<!-- Do not change. -->
+To be completed before the issue can be marked as Done.
+- [ ] Meets acceptance criteria
+- [ ] Complies with the standard of public code
+- [ ] Story has been sufficiently tested
+    - [ ] 90% code coverage
+    - [ ] Build completed
+    - [ ] Existing tests pass
+- [ ] The frontend component complies with [WCAG 2.1 accessibility guidelines](https://www.w3.org/TR/WCAG21/)
```
Comment (bddjong) [inline .github/ISSUE_TEMPLATE/feature_request.md:44]: Front-end component is reviewed by designers in storybook

**Review by bddjong — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (GewoonMaarten) [reply .github/ISSUE_TEMPLATE/feature_request.md]: It was meant as an example

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+A clear and concise description of what the bug is.
+
+### To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected behavior
```
Comment (bddjong) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Maybe add a section with the current behavior as well

Comment (GewoonMaarten) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: Isn't that the "Describe the bug" section?

**Review by bddjong — DISMISSED** [review]

**Comment by mjcarsjens** [comment]
Nice, good call to add these!

**Review by DanielvanVliet — APPROVED** [review]

**Pull request closed without merging**